spacy/tokens/_retokenize.pyx

```diff
@@ -480,3 +505,13 @@ def _validate_extensions(extensions):
             raise ValueError(Errors.E118.format(attr=key))
         if not is_writable_attr(extension):
             raise ValueError(Errors.E119.format(attr=key))
+
+
+cdef make_iob_consistent(TokenC* tokens, int length):
+    cdef int i
+    if tokens[0].ent_iob == 1:
+        tokens[0].ent_iob = 3
+    for i in range(1, length):
+        print(tokens[i].ent_iob, tokens[i].ent_type)
```

Comment:
```suggestion

```

Reply:
You mean you don't want my debugging code?!?